pkg/es/config/config.go

```diff
@@ -115,6 +115,14 @@ type Configuration struct {
 	// Use this option with Elasticsearch rollover API. It requires an external component
 	// to create aliases before startup and then performing its management.
 	UseReadWriteAliases bool `mapstructure:"use_aliases"`
+	// ReadAliasSuffix is the suffix to append to the index name used for reading.
+	// This configuration only exists to provide backwards compatibility for jaeger-v1
+	// which is why it is not exposed as a configuration option for jaeger-v2
```

Comment:
if we don't expose this for v2 config, how would it be backwards compatible with the existing index names?

Reply:
@yurishkuro we already took care of that in https://github.com/jaegertracing/jaeger/pull/6519. We made it a breaking change and provided instructions to the user on how to migrate over.

Reply:
nice!